Makefile

```diff
@@ -641,3 +641,8 @@ test/scripts:
 	# Preform a basic check on scripts checking for a non zero exit
 	echo "Running make release/prepare"
 	SEMVER=$(RHOAM_TAG) make release/prepare
+
+.PHONY: install-oo
+install-oo:
+	@echo "Installing Cluster Observability Operator Subscription..."
```

Comment:
Creating Cluster Observability Operator Subscription

It's installed by OLM

Reply:
Updated. Thank you